src/hooks/useUsageCalc.test.tsx

```diff
@@ -67,16 +84,18 @@ describe('http usage', () => {
       probes: [1],
       settings: {
         http: {
-          tlsConfig: {},
+          tlsConfig: {
+            serverName: `trigger SSL`,
+          },
         } as HttpSettings,
       },
       frequency: 10000,
-    });
+    } as Check);
     expect(withSSL.current).toStrictEqual({
       checksPerMonth: 262800,
-      activeSeries: 118,
+      activeSeries: http_ssl.Series,
       logsGbPerMonth: 0.21,
-      dpm: 708,
```

Comment:
In the test all of the references to SSL assertions had identical values to the non-SSL entries which seems to be wrong?